docs/resources/cce_node.md

```diff
@@ -244,6 +330,51 @@ extend_param = {
   + `effect` - (Required, String, ForceNew) Available options are NoSchedule, PreferNoSchedule, and NoExecute.
     Changing this parameter will create a new resource.
 
+The `selectors` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the selector name, used as the index of `selector_names` in storage group.
+  The name of each selector must be unique. Changing this parameter will create a new resource.
+* `type` - (Required, String, ForceNew) Specifies the storage type. Currently, only **evs (EVS volumes)** is supported.
+  Changing this parameter will create a new resource.
```

Comment:
suggest to change it to be optional if it only supports evs type.

Reply:
sure, fixed